src/main/java/com/artipie/api/RepositoryRest.java

```diff
@@ -73,28 +86,35 @@ public void init(final RouterBuilder rbr) {
      */
     private void getRepo(final RoutingContext context) {
         final RepositoryName rname = new RepositoryName.FromRequest(context, this.layout);
-        final ValidRepositoryName validator = new ValidRepositoryName(rname);
-        if (!validator.isValid()) {
-            context.response()
-                .setStatusCode(HttpStatus.BAD_REQUEST_400)
-                .end(validator.errorMessage());
-            return;
-        }
-        if (!this.crs.exists(rname)) {
-            context.response()
-                .setStatusCode(HttpStatus.NOT_FOUND_404)
-                .end(String.format("Repository %s does not exist. ", rname));
-            return;
-        }
-        if (this.crs.hasSettingsDuplicates(rname)) {
+        final RepositoryNameCondition rnvalidator = new RepositoryNameCondition(rname);
+        final Validator validator = validator(
+            validator(
+                rnvalidator,
+                rnvalidator,
+                HttpStatus.BAD_REQUEST_400
+            ),
+            validator(
+                () -> this.crs.exists(rname),
+                () -> String.format("Repository %s does not exist.", rname),
+                HttpStatus.NOT_FOUND_404
+            ),
+            validator(
+                () -> !this.crs.hasSettingsDuplicates(rname),
+                () -> String.format(
+                    new StringBuilder()
```

Comment:
Suggest make one string and use `// @checkstyle LineLengthCheck (1 line)`

Reply:
@dgarus Good advices, thanks!
Added changes, @dgarus and @olenagerasimova please review again.